### DISCUSSION TITLE
Train Stop Parameters

### DISCUSSION THREAD

**Discussion by Redfog1** [Opinionated-Blueprints/10-Books-Full-of-Rails#21] [Q&A] [answered]
I was wondering what the Group and Item Parameters do within the train Requester and Provider arrays. just started using the book and would love the clarification. that being said, this is beautiful work with the print book.

**Comment by Bocian-1** [comment, marked as answer]
Groups: https://github.com/Opinionated-Blueprints/10-Books-Full-of-Rails/wiki/Groups
The rest: https://github.com/Opinionated-Blueprints/10-Books-Full-of-Rails/wiki/Provider-and-Requester-stations
